code/modules/paperwork/paper.dm

```diff
@@ -175,7 +175,7 @@
 /obj/item/paper/ui_state(mob/user)
 	if(istype(loc, /obj/item/modular_computer))
 		return GLOB.reverse_contained_state
-	return ..()
+	return logic_state(STATE_OR, list(..(), GLOB.observer_state))
```

Comment:
This will allow ghosts to write to paper via a TGUI exploit assuming the UI doesn't allow them to do that already

Reply:
Unless observer state doesn't allow for interactions?